taskt/Core/Server/HttpServerClient.cs

```diff
@@ -86,40 +86,41 @@ public static bool GetGuid(bool overrideExisting = false)
         {
             try
             {
-                if (!(appSettings.ServerSettings.ServerConnectionEnabled))
+                if (!_appSettings.ServerSettings.ServerConnectionEnabled)
                     return false;
 
-                httpLogger.Information("Client attempting to retrieve a GUID from the server, override " + overrideExisting);
-
-
-
-
+                _httpLogger.Information("Client attempting to retrieve a GUID from the server, override " + overrideExisting);
 
-                if (appSettings.ServerSettings.HTTPGuid == Guid.Empty || overrideExisting)
+                if (_appSettings.ServerSettings.HTTPGuid == Guid.Empty || overrideExisting)
                 {
-                    httpLogger.Information("Requesting GUID from the application server");
+                    _httpLogger.Information("Requesting GUID from the application server");
                     var client = new WebClient();
 
                     var userName = Environment.UserName;
                     var machineName = Environment.MachineName;
-                    var content = client.DownloadString(appSettings.ServerSettings.HTTPServerURL + "/api/Workers/New?userName=" + userName + "&machineName=" + machineName);
 
-                    httpLogger.Information("Received /api/Workers/New response: " + content);
-                    var deserialized = Newtonsoft.Json.JsonConvert.DeserializeObject<Worker>(content);
+                    string uri = _appSettings.ServerSettings.HTTPServerURL +
+                        "/api/Workers/New?userName=" + userName +
+                        "&machineName=" + machineName;
 
-                    appSettings.ServerSettings.HTTPGuid = deserialized.WorkerID;
-                    new ApplicationSettings().Save(appSettings);
+                    var content = client.DownloadString(uri);
+
+                    _httpLogger.Information("Received /api/Workers/New response: " + content);
+                    var deserialized = JsonConvert.DeserializeObject<Worker>(content);
+
+                    _appSettings.ServerSettings.HTTPGuid = deserialized.WorkerID;
+                    new ApplicationSettings().Save(_appSettings);
                     return true;
                 }
                 else
                 {
-                    httpLogger.Information("Client has an assigned GUID or override existing GUID set to false. To retrieve a new GUID, the existing setting must be cleared.");
+                    _httpLogger.Information("Client has an assigned GUID or override existing GUID set to false. To retrieve a new GUID, the existing setting must be cleared.");
```

Comment:
Too long maybe?

Reply:
Agree'd, made the change and split it up.